static/js/zamboni/buttons.js

```diff
@@ -403,10 +403,12 @@ var installButton = function() {
         var opts = search ? {addPopup: false, addWarning: false} : {};
         versionsAndPlatforms(opts);
     } else if (z.app == 'firefox') {
+        $('#downloadAnyway').attr('href',escape_($button.filter(':visible').attr('href')));
```

Comment:
Is the `.filter(':visible')` necessary? Wouldn't `$button.find('.installer').attr('href')` do the work?